### DISCUSSION TITLE
Is it possible to install Cairo in Linux Mint Cinnamon, and what is required?

### DISCUSSION THREAD

**Discussion by ineuw** [Cairo-Dock/cairo-dock-core#55] [Q&A] [answered]
I am using LMC 22.0 and wish to use Cairo, but I am completely clueless as to what is needed. Are there any instructions? For example, the Linux Mint repository lists numerous packages but without any information about the basic requirements.

**Comment by dkondor** [comment, marked as answer]
Please see here for the basic info:
https://github.com/Cairo-Dock/cairo-dock-core/wiki/Installation

You will want the following packages:
 - `cairo-dock`: this should install all main dependencies, including plugins
 - `cairo-dock-gnome-session`: this adds Cairo-Dock to your login menu (as a custom session), which is the recommended way to run it on X11; however, this is optional and you could just add Cairo-Dock to your autostart (see [here](https://github.com/Cairo-Dock/cairo-dock-core/wiki/Running) )

It is recommended to install from one of our PPAs as the packages in the official repositories are outdated:
https://launchpad.net/~cairo-dock-team/+archive/ubuntu/ppa
https://launchpad.net/~cairo-dock-team/+archive/ubuntu/weekly

Let us know if you run into issues or have further questions!

**Reply by ineuw** [reply]
Thanks for the info. I installed it.
